assets/js/admin-csv-import.js

```
@@ -260,3 +260,57 @@ jQuery(function( $ ) {
+
+const addFileSizeError = (input) => {
+    const parentElements = getParentElements(input);
+
+    parentElements.label.style.color = 'red';
+
+    const error = document.createElement('p');
+    error.classList.add('wpbdp-csv-import-error');
+    error.textContent = window.wpbdp_admin_import.error_label;
+    parentElements.labelWrapper.appendChild(error);
+
+    document.querySelectorAll('.submit input[type=submit]').forEach((el) => el.disabled = true);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)